spec/active_record/typed_store_spec.rb

```diff
@@ -14,6 +14,14 @@
 
   end
 
+  describe 'Marshal.dump' do
+
+    it 'dumps the model' do
+      Marshal.dump(model)
+    end
```

Comment:
It seems very sketchy to me that marshalling an activerecord instance also marshal the coder which is supposed to stay a class attribute.

I'll hold on this a little bit to figure out the root cause if you don't mind.